extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/QuteProcessor.java

```diff
@@ -821,7 +821,20 @@ public boolean test(String name) {
             }
         });
 
-        ValueResolverGenerator.Builder builder = ValueResolverGenerator.builder().setIndex(index).setClassOutput(classOutput);
+        List<DotName> entitySuperClasses = new ArrayList<>();
+        entitySuperClasses.add(DotName.createSimple("io.quarkus.hibernate.orm.panache.PanacheEntity"));
+        entitySuperClasses.add(DotName.createSimple("io.quarkus.hibernate.orm.panache.PanacheEntityBase"));
+        entitySuperClasses.add(DotName.createSimple("io.quarkus.mongodb.panache.PanacheMongoEntity"));
+        entitySuperClasses.add(DotName.createSimple("io.quarkus.mongodb.panache.PanacheMongoEntityBase"));
+        ValueResolverGenerator.Builder builder = ValueResolverGenerator.builder()
+                .setIndex(index)
+                .setForceGettersPredicate(new Predicate<ClassInfo>() {
+                    @Override
+                    public boolean test(ClassInfo clazz) {
+                        DotName superName = clazz.superName();
+                        return superName != null && entitySuperClasses.contains(superName);
```

Comment:
This is forgetting HR with Panache, mongo reactive, and kotlin versions. Also this only tests for direct superclasses, but there are entire hierarchies of such entities. I fear this is going to be too fragile. Perhaps we should either use a panache build item or make a new build item in core for this sort of case?

Reply:
We could use the `PanacheEntityClassesBuildItem` if we create an SPI module. I'm not so sure about moving this build item to the core...

Reply:
SPI module is fine, but perhaps the `-common` module is already an SPI?

Reply:
Hm, I can try...

Reply:
It seems to work :-)